lib/hanami/action.rb

```diff
@@ -440,7 +441,7 @@ def session_enabled?
     # @since 2.0.0
     # @api private
     def build_request(**options)
-      Request.new(**options)
+      Request.new(default_tld_length: config.default_tld_length, **options)
```

Comment:
Whoops, I gave you bad guidance on this one. This arg should be passed down in `#call`, as one of these:

```ruby
        request  = build_request(
          env: env,
          params: params,
          session_enabled: session_enabled?
        )
```